### ISSUE TITLE
Add documentation and helpful links for downloading images

### ISSUE THREAD

**Issue by yeazelm** [bottlerocket-os/bottlerocket-project-website#216]
**Path to page with content (_do not_ include domain name):** https://bottlerocket.dev/en/os/1.14.x/install/

**What content is incorrect?** It only sort of covers how to use the quickstarts. But for metal, I have to work through https://github.com/bottlerocket-os/bottlerocket/blob/develop/PROVISIONING-METAL.md#fetch-the-bottlerocket-image-for-bare-metal which has a lot of steps, including installing Rust and compiling `tuftool` from source. This involves multiple steps when all I want to do is download the image and try it on my hardware. See https://github.com/bottlerocket-os/bottlerocket/discussions/2779 for a deeper discussion on how this could be improved.

**What should the content say or communicate?**
Ideally, I would like to see as simple as possible process to get an image on my local machine. Many Linux distros have download links for their images. A list of URLs for each variant and version where I could click the button and download the image would be easiest. I would probably want a SHA sum next to the URL provided so I could verify I got the right image and its intact once downloaded.

An alternative that would still make things a lot better would be to provide a pre-build binary of `tuftool` (to avoid having to install Rust and then compile `tuftool`) and the website provide the full invocation for `tuftool` so the "hard work" is done for me of figuring out things like "metal-k8s-1.27" is what I want for the url for variant or what the latest version is at that moment in time.

**Comment by stockholmux** [comment]
You're right. I think setting up Bottlerocket has several side quests. Not ideal. The website could be used to smooth the rough edges here; it's a substantial UX surface area.

The click-download-install workflow is somewhat of a user expectation for software which Bottlerocket doesn't have. 

How folks feel about the implications of downloading w/o verification by tuftool is another question. @bcressey @etungsten @zmrow do you have thoughts on this?

I think all the currently supported platforms would need to be listed on a "downloads" page just so folks don't go down a confusing path thinking they need to download an image for EC2 or something. 

[As for the alternative, making a pre-built binary of tough is out of scope for the website - but perhaps not the project at large - but even giving a straight-forward, copy-pastable way to use tuftool would be helpful.]
